UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/ControlAction.cs

```diff
@@ -12,13 +12,40 @@ public class ControlAction : MonoBehaviour
 
 		public Image pullImage;
 
+		public static bool ThrowHold = true;
+
 		private void Start()
 		{
 			UIManager.IsThrow = false;
 
 			pullImage.enabled = false;
+			ThrowHold = GetThrowPreference();
+		}
+
+		public static bool GetThrowPreference()
+		{
+			return 1 == PlayerPrefs.GetInt("ThrowHoldPreference", 1);
 		}
 
+
+		public static void SetPreference(bool preference)
```

Comment:
change the name to ``SetThrowHoldPreference`` otherwise, the name is way too generic and can cause confusion on other developers when reading it.